db/migrations/0001_initial.py

```diff
@@ -1,44 +1,58 @@
-# Generated by Django 4.0.2 on 2022-08-19 07:39
+# Generated by Django 4.0.2 on 2023-02-13 21:37
 
 from django.db import migrations, models
+import django.db.models.deletion
```

Comment:
We are making reviews not only on particular commit, but for all changes that are present on the PR. Why did you duplicate migrations in separate folder?

Reply:
That's becuse of refusing to do PR. Just only tried to find why i cannot create PR

Reply:
not fixed

Reply:
I've made new migrations and deleted last ones